### PR TITLE
Coverall file reduction

### PR DESCRIPTION
**Pull request by austincunningham**
## Overview
Remove deepcopy and other auto generated files from tests
Jira: https://issues.jboss.org/browse/INTLY-4041

## Verification

- Clone this branch
- Run `make test/unit/ci`
- Run `make test/unit/coverage`
- review the web and cli reports coverage should be about 57%

### CODE REVIEW

File: pkg/providers/openshift/provider_postgres.go [hunk 1]
```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	types2 "github.com/integr8ly/cloud-resource-operator/pkg/apis/integreatly/v1alpha1/types"
```
[the remark this answers is not in the document]
Comment (austincunningham) [reply pkg/providers/openshift/provider_postgres.go]: golang refactor

**Comment by ciaranRoche** [comment]
Tests are passing now, are we going to change the naming convention to all be `types` or are we leaving it as it is?

**Comment by austincunningham** [comment]
@ciaranRoche going to change it as its not needed.

**Comment by austincunningham** [comment]
@ciaranRoche actually is need as it has a clash with k8 types here
https://github.com/integr8ly/cloud-resource-operator/pull/41/files?file-filters%5B%5D=.go&file-filters%5B%5D=No+extension#diff-23edd8ba0dba05c5dec15ae2c0a3082eR25
So will leave it.

**Pull request merged**